user_docs/en/changes.md

```diff
@@ -11,6 +11,7 @@
 
 ### Changes
 
+* Updated fast_diff_match_patch to version 2.1.0. (#16508, @codeofdusk)

```

Comment:
I think we need a user oriented explanation if this is in changes, otherwise it should go to changes for developers

```
suggestion
* Updated terminal processing library fast_diff_match_patch to version 2.1.0. (#16508, @codeofdusk)
```